docs/application-connector/docs/06-01-application.md

```diff
@@ -53,6 +53,16 @@ This table lists all the possible parameters of a given resource together with t
 | **spec.services.entries.oauthUrl** |    **NO**   | Specifies the URL used to authorize with a given API. This field is required for the API entry type. Parameter provided by the Application Registry, do not edit. |
 | **spec.services.entries.credentialsSecretName** |    **NO**   | Specifies the name of the Secret which allows you to call a given API. This field is required if **spec.services.entries.oauthUrl** is specified. Parameter provided by the Application Registry, do not edit. |
 
+## Related resources and components
+
+These components use this CR:
+
+| Component   |  Description |
+|:-----------:|:-------------|
+| Application Registry | Reads and writes the APIs and Event Catalog metadata of the connected external solution to this CR. |
```

Comment:
```suggestion
| Application Registry | Reads and writes the APIs and Event Catalog metadata of the external solution connected to this CR. |
```

Reply:
An external solution doesn't connect to the CR, it connects to Kyma through the Application Connector.